src/index.ts

```diff
@@ -1,4 +1,36 @@
-//Uncomment to see the choose pokemon page
-//const newTemplate = document.querySelector('#choose-page-template')!;
-//const app = document.querySelector('#app')!;
-//app.innerHTML = newTemplate.innerHTML
\ No newline at end of file
+import { showAModal, hideAModal } from "./modalPopUpFunctions";
+
+const renderStartingPage = (appDiv: HTMLDivElement) => {
```

Comment:
minor thing, don't need to change: why naming "appDiv" when type tells you that it is "HTML**Div**Element"? :P